tasks/sep/007-presigned-pause/.env

```diff
@@ -0,0 +1,5 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+DEPUTY_GUARDIAN_MODULE_ADDR=0xed12261735aD411A40Ea092FF4701a962d25cA21
+SUPERCHAIN_CONFIG_ADDR=0xC2Be75506d5724086DEB7245bd260Cc9753911Be
+OWNER_SAFE=0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B
```

Comment:
Can we rename `OWNER_SAFE` to something more clear? The proxy admin owner is 0x1Eb2fFc903729a0F03966B917003800b145F56E2, and this 0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B address is a signer on that safe

Reply:
Using PRESIGNER_SAFE for now.